nixos/modules/services/networking/nebula.nix

```diff
@@ -231,6 +249,10 @@ in
               ''
               settings
           );
+          capabilities = lib.concatStringsSep " " (
+            (lib.optional (!settings.tun.disabled) "CAP_NET_ADMIN")
```

Comment:
You can get rid of the parens around (lib.optional ..) - ++ is higher precedence than the function call.

Reply:
Thank you, TIL